pulp_container/tests/functional/api/test_push_signatures.py

```diff
@@ -0,0 +1,120 @@
+"""Tests that verify that an image signature can be pushed to Pulp."""
+import gnupg
+import base64
+import json
+import os
+import re
+import requests
+import subprocess
+import pytest
+
+from pulp_smash.pulp3.bindings import (
+    delete_orphans,
+    monitor_task,
+)
+from pulp_container.tests.functional.constants import REGISTRY_V2_REPO_PULP
+from pulp_container.tests.functional.utils import get_auth_for_url
+
+KEY = "https://raw.githubusercontent.com/pulp/pulp-fixtures/master/common/GPG-PRIVATE-KEY-pulp-qe"
+
+gpg = gnupg.GPG()
+
+
+@pytest.fixture
+def key_data():
+    """Return the ID and fingerprint of the Pulp QE GPG key."""
+    private_key = requests.get(KEY).text
```

Comment:
I think we should use the tmpdir fixture here. (Not sure if it allows session scope.)
https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture

Reply:
I am receiving: `ScopeMismatch: You tried to access the function scoped fixture tmpdir with a module scoped request object`. So, I will not use `tmpdir` and leave it as it is.